query/src/storages/fuse/fuse_history.rs

```diff
@@ -82,23 +89,25 @@ impl<'a> FuseHistory<'a> {
             current_snapshot_version = ver;
         }
 
-        DataBlock::create(FuseHistory::schema(), vec![
+        Ok(DataBlock::create(FuseHistory::schema(), vec![
             Series::from_data(snapshot_ids),
-            Series::from_data(prev_snapshot_ids),
+            Series::from_data(snapshot_locs),
```

Comment:
I think the column  name `snapshot_locs` > `snapshot_locations` is more clearly for the user.

Reply:
OK